util.js

```diff
@@ -30,7 +26,7 @@ exports.tarHeaderChecksumMatches = buffer => { // Does not check if checksum fie
 		signedBitSum += byte & MASK_8TH_BIT; // Add signed bit to signed bit sum
 	}
 
-	const readSum = parseInt(uint8ArrayUtf8ByteString(buffer, 148, 154), 8); // Read sum in header
+	const readSum = parseInt(buffer.toString('binary', 148, 154), 8); // Read sum in header
```

Comment:
Curious why the encoding parameter is binary here? At first glance this seems like it's not a 1:1 replacement

Reply:
Correct, it is not. I don't think UTF is the proper encoding: [_"All characters in header blocks are represented by using 8-bit characters in the local variant of ASCII"_](https://www.gnu.org/software/tar/manual/html_node/Standard.html). Having another look to it, neither `binary` (this is an alias for `ISO8859-1`), I will set it to `ascii`. It should not make a difference at the end, in case it is a tar file, only text encoded digits are expected, which are all encoded the same way in these encoding types.